src/estimation/src/BerdyHelper.cpp

```diff
@@ -1526,6 +1589,20 @@ bool BerdyHelper::updateKinematicsFromFloatingBase(const JointPosDoubleArray& jo
     m_jointPos = jointPos;
     m_jointVel = jointVel;
 
+    // Update kinDyn computation
+    iDynTree::KinDynComputations kinDynComputations;
+    kinDynComputations.loadRobotModel(m_model);
+    kinDynComputations.setRobotState(m_jointPos,
+                                     m_jointVel,
+                                     m_gravity);
+
+    // TODO: Update base_H_m_links transformations
```

Comment:
TODO can be removed I suppose?

Reply:
Done in https://github.com/robotology/idyntree/pull/571/commits/f57ab3c061b91f83c5351d671132cc922ca75556